oak-store-document/src/main/java/org/apache/jackrabbit/oak/plugins/document/mongo/MongoDocumentNodeStoreBuilderBase.java

```diff
@@ -104,18 +103,11 @@ public T setMongoDB(@NotNull MongoClient client,
      * @param enable whether to enable or disable it.
```

Comment:
Can you please update JavaDoc and mention this is deprecated and calling this method does not have any effect? Socket keep-alive is always enabled.